lib/src/lib.rs

```diff
@@ -33,6 +33,48 @@ pub mod protocol_instance;
 pub mod prover;
 pub mod taiko_utils;
 
+#[cfg(not(target_os = "zkvm"))]
+mod time {
+    pub use core::ops::AddAssign;
+    pub use std::time::{Duration, Instant};
+}
+#[cfg(target_os = "zkvm")]
```

Comment:
Is there issues if we use std::time in zkvm?

Reply:
Yep! Halts in some time syscall.